xp0-double/xp0d-azuredeploy-msdeploy.json

```diff
@@ -190,9 +190,7 @@
                   "Application Insights Instrumentation Key": "[reference(concat('Microsoft.Resources/deployments/', 'sitecore-infra'), '2015-11-01').outputs.appInsightsInstrumentationKey.value]",
                   "Application Insights Role": "CM",
                   "KeepAlive Url": "[concat('https://', reference(resourceId('Microsoft.Web/sites', variables('cmWebAppNameTidy'))).hostNames[0], '/sitecore/service/keepalive.aspx')]",
-                  "License Xml": "[variables('licenseXml')]",
-                  "IP Security Client IP": "[parameters('security_clientIp')]",
```

Comment:
sorry, yes, as an example, it does.. but I assume one runs them from their own repo or fork, so don't have to pull back any changed settings back into this - all your other additions and remarks are very welcome and I would like to update my demo set with them, but this one especially should either be removed on your side, or added within your SCWDP